components/devtools_traits/lib.rs

```diff
@@ -83,6 +83,7 @@ pub enum ScriptToDevtoolsControlMsg {
     ),
     /// A particular page has invoked the console API.
     ConsoleAPI(PipelineId, ConsoleMessage, Option<WorkerId>),
+    PageErrorAPI(PipelineId, PageErrorAPI, Option<WorkerId>),
```

Comment:
Add `/// An unhandled exception has occurred in a particular page.` Also I think `PageError` is more clear; the previous message is `ConsoleAPI` because it is a message for the methods of the Console API.